biz.aQute.bndlib/src/aQute/bnd/service/reporter/BundleEntryPlugin.java

```diff
@@ -0,0 +1,41 @@
+package aQute.bnd.service.reporter;
+
+import java.util.Locale;
+
+import org.osgi.annotation.versioning.ProviderType;
+
+import aQute.bnd.osgi.Jar;
+import aQute.bnd.osgi.Processor;
+
+/**
+ * This plugin extracts a piece of information from a Jar and converts it into a
+ * DTO representation. Its result will contributes to the formation of a Jar
+ * metadata report.
+ */
+@ProviderType
+public interface BundleEntryPlugin {
+
+	/**
+	 * Extracts a piece of information from the Jar in arguments.
+	 * <p>
+	 * If the Jar contains localized data, it will be extracted for the
+	 * specified locale or a less specific if not found. The syntax of the
+	 * locale must be as specified in {@link Locale}, an empty {@code String}
+	 * means {@code unlocalized}.
+	 * 
+	 * @param jar the Jar to inspect, must not be {@code null}
+	 * @param locale the {@code String} representation of a {@code Locale}, must
```

Comment:
Shouldn't this param be of type `Locale`?

Reply:
Why don't the extract methods of the project and workspace plugins also need locale params?

Reply:
> Shouldn't this param be of type Locale?

This is because, as far as I remember, a Local object cannot represent an unlocalized form. So in the case where user is not interested in localized data, the argument as to be null. But I think we can switch if you prefer.

> Why don't the extract methods of the project and workspace plugins also need locale params?
I know that bundle defines standard way to provide localized data (manifest, metatype) but not for projects or workspaces!?